### DISCUSSION TITLE
Is there any Grafana dashboard I can use for the sql_exporter?

### DISCUSSION THREAD

**Discussion by Neaj-Morshad-101** [burningalchemist/sql_exporter#533] [Q&A] [answered]
Could you let me know if any developed dashboards are available for this sql_exporter? 

Like for [prometheus-mssql-exporter](https://github.com/awaragi/prometheus-mssql-exporter), There is a [grafana dashboard](https://grafana.com/grafana/dashboards/13919-microsoft-sql-server/) developed.

I would greatly appreciate any suggestions and feedback you might have about this.

**Comment by burningalchemist** [comment, marked as answer]
Hi @Neaj-Morshad-101, I'd say given the arbitrary nature of `sql_exporter` in terms of database engines and overall metrics (business or infra), it'd be difficult to come up with certain dashboards.

There is a community contribution you might find here - https://github.com/burningalchemist/sql_exporter/tree/master/examples/azure-sql-mi/grafana-dashboard (for azure_sql). But I'd leave it to community to develop and contribute any examples.

The way I see it is to get an available dashboard from Grafana library or Github, and adjust for your needs. I'm open to PRs though, so in the future we might host the example dashboards on the docs webpage.
